src/welcome/welcome.ts

```diff
@@ -35,13 +37,49 @@ export class WelcomePageProvider extends Disposable implements TextDocumentConte
         return this._context.globalState.get<boolean>(GAUGE_SUPPRESS_WELCOME);
     }
 
+    getLinuxDistribution(): string {
+        let dist = spawnSync('cat', ['/etc/issue']);
+        if (dist.error) {
+            return null;
+        }
+        return dist.stdout.toString();
+    }
+
+    getInstallCommandBasedOnOS(): any {
+        let installCommand: any = {};
+        switch (process.platform) {
+            case "win32":
+                installCommand.name = "Chocolatey";
+                installCommand.command = "choco install gauge";
+                return installCommand;
+            case "darwin":
+                installCommand.name = "Brew";
+                installCommand.command = "brew install gauge";
+                return installCommand;
+            default:
+                if (this.getLinuxDistribution().indexOf("Ubuntu") !== -1) {
+                    installCommand.name = "apt-get";
```

Comment:
debian package manager is `apt`. 

Also lets be consistent on the casing. `Brew` and `Chocolatey` have caps in the name, `apt` and `dnf` do not.

Consider making them `brew`, `choco`, `apt` and `dnf`